controllers/grid/users/stageParticipant/form/PKPStageParticipantNotifyForm.php

```diff
@@ -235,10 +234,10 @@ public function sendMessage(int $userId, Submission $submission, Request $reques
         );
 
         $mailable->allowUnsubscribe($notification);
-        $logDao = null;
+        $log = null;
         try {
             Mail::send($mailable);
-            $logDao = DAORegistry::getDAO('SubmissionEmailLogDAO'); /** @var SubmissionEmailLogDAO $logDao */
+            $log =  Repo::emailLogEntry();
```

Comment:
Call this `$logRepository` rather than `$log`; at first glance I thought this was a log entry.

Reply:
update